packages/components/src/components/pagination/_pagination.scss

```diff
@@ -42,6 +42,19 @@ $css--helpers: true;
     @include carbon--breakpoint('md') {
       overflow: initial;
     }
+
+    // mobile friendly pagination
+    @include carbon--breakpoint-down('md') {
+      .#{$prefix}--pagination__left > *,
+      .#{$prefix}--pagination__right > * {
+        display: none;
```

Comment:
Not sure we want to simply hide these items. We'd need to leave the buttons to at least allow navigating to different pages.

Reply:
I don't think there is too much that can fit into the smallest viewport supported (iPhone, 320 px). I can check how it looks like on that width when the page selector is visible and attach a screenshot.

Reply:
It actually might just work if I remove the margin on the left, of course would only cover english so we might have issues using other languages.

<img width="353" alt="Screenshot 2020-07-29 at 13 53 49" src="https://user-images.githubusercontent.com/6061509/88797084-0ad7a300-d1a3-11ea-86c4-4de5893b419f.png">

Reply:
I was actually wrong, I couldn't make it work when the text is longer

<img width="354" alt="Screenshot 2020-07-29 at 14 21 53" src="https://user-images.githubusercontent.com/6061509/88799543-f0072d80-d1a6-11ea-93f5-48a0fcc6b545.png">

Reply:
Your solution works how I was expecting, using the page buttons and hiding the page selector dropdown on mobile. As long as users have a way to navigate through pages we're good 👍